README.md

```diff
@@ -17,6 +17,12 @@ SDK for interacting with Joyent's Manta system.
 ### Requirements
 * [Java 1.8](http://www.oracle.com/technetwork/java/javase/downloads/index.html) or higher.
 * [Maven 3.3.x](https://maven.apache.org/)
+* [Java Cryptography Extension](http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html)
+
+#### Add [BouncyCastle](http://www.bouncycastle.org/latest_releases.html) as a security provider
```

Comment:
After installing JSE I had to also add bouncy castle to the list of trusted providers